docs/tutorials/first_app/blinky_compiler.md

```diff
@@ -1,70 +1,81 @@
-## Blinky on the Arm Mbed Online Compiler
+## Mbed Online Compiler
 
-This tutorial builds Blinky using the Arm Mbed Online Compiler, which allows you to build Mbed OS applications without installing a toolchain on your own machine.
+### Setup
 
-<span class="tips">Please create a <a href="https://os.mbed.com/account/signup/" target="_blank">developer account</a>. It's free, and we don't spam.</span>
+#### Create an Mbed developer account
 
-### Importing Blinky
+Go to [os.mbed.com](os.mbed.com), and [create an account](https://os.mbed.com/account/signup/F).
 
-To get Blinky into the Mbed Online Compiler, click the **`Import into Mbed IDE`** button below:
+#### Setup environment
 
-[![View code](https://www.mbed.com/embed/?url=https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-blinky/)](https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-blinky/file/tip/main.cpp)
+- Plug your Mbed board into your computer, and open its USB device folder.
+- Double click on the `MBED.HTM` file. (This adds your Mbed platform to the Online Compiler.)
 
-You're taken to the online IDE, and the **Import Program** dialog box opens:
+If you do not have an Mbed board, go to [os.mbed.com/platforms](http://os.mbed.com/platforms), select a board and click the “Add to your Mbed Compiler” button.
+<span class="images">![](https://sarahmarshy.github.io/img/add-to-compiler.png)
+</span>
 
-<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/import_dialog.png)<span>Importing Blinky</span></span>
+### Code
 
-The import mechanism offers a default name, but you're free to change it. When you're done, click **Import**.
+#### Import 
 
-### Viewing Blinky
+Visit the Mbed OS [blinky example repository](https://developer.mbed.org/teams/mbed-os-examples/code/mbed-os-example-blinky/), and click the "Import into Compiler" button.
+ 
+<span class="images">![](https://sarahmarshy.github.io/img/import-compiler.png)
```

Comment:
This image is out of date. Please update it and upload it to the "images" folder at a higher level of the Handbook.

Reply:
@yennster can you please update the image?
@AnotherButler where would you like us to upload the image? should we create a `img` folder or is there a master image folder we should upload to?

Reply:
As mentioned previously, there's an "images" folder at a high level in our documentation where we put all images: https://github.com/ARMmbed/Handbook/tree/new_engine/docs/images